internal/async/map_migratedfynedo.go

```diff
@@ -0,0 +1,71 @@
+//go:build migrated_fynedo
+
+package async
+
+// Map is a generic wrapper around [sync.Map].
+type Map[K any, V any] struct {
+	// once go1.20 is base, can use map[K]V
+	// with K being Comparable instead of any
+	// (CanvasObject, etc aren't Comparable for go1.19)
+	m map[any]V
```

Comment:
This should be [K]V, right?

Reply:
Ah, I missed the comment. Sorry. Might it make sense to instead just remove the methods of the map that require them to be comparable? I don’t think they are used in too many places for it to matter. Should be a good performance boost.

Reply:
Or is it the amp itself that complains because it can't hash the values?